tagless-final/build.gradle

```diff
@@ -72,10 +72,8 @@ dependencies {
   compile "com.squareup.picasso:picasso:2.5.2"
 
   kapt "io.arrow-kt:arrow-annotations-processor:$arrow_version"
-  compile "io.arrow-kt:arrow-core:$arrow_version"
-  compile "io.arrow-kt:arrow-typeclasses:$arrow_version"
-  compile "io.arrow-kt:arrow-instances:$arrow_version"
-  compile "io.arrow-kt:arrow-effects:$arrow_version"
+  kapt "com.google.dagger:dagger-compiler:$dagger_version"
```

Comment:
Why removing the core? following the docs, the core is needed. If it's being synced transitively I feel like the docs could be missleading since they look like you should import the artifacts separately and always optionally.

Reply:
arrow-dagger depends on core. Restating dependencies that are already brought transitively is weird but feel free to change it commiting to this branch,  I don't think explicitly declaring them brings any value. Also I don't like how the optional / required stuff is in the docs. It should just list deps per module.